Gemfile

```diff
@@ -30,7 +30,7 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
 
-gem 'formtastic'
+gem 'simple_form'
```

Comment:
You could mention that change in the PR description :-) I know why that's happened and I was expecting that, but in reality you do not always work that close with all other programmers in the team :+1: